src/Stripe.net/Entities/Invoices/Invoice.cs

```diff
@@ -157,6 +157,14 @@ internal object InternalCustomer
         [JsonProperty("customer_shipping")]
         public Shipping CustomerShipping { get; set; }
 
+        /// <summary>
+        /// The customer’s tax exempt status. Until the invoice is finalized, this field will equal
+        /// customer.tax_exempt. Once the invoice is finalized, this field will no longer be
```

Comment:
```suggestion
        /// <see cref="Customer.TaxExempt" />. Once the invoice is finalized, this field will no longer be
```

Reply:
We did not do this for the other examples/docs in that file. That also will likely never work with autogen to be honest.